cucumber/api/features/step_definitions/logs_steps.rb

```diff
@@ -1,17 +1,13 @@
 # frozen_string_literal: true
 
 Given(/^I save the log data from bookmark$/) do
-  lines_amount_in_bookmark()
-end
-
-Given(/^I save the log data from bookmark "([^"]*)"$/) do |bookmark|
-  lines_amount_in_bookmark(bookmark)
+  save_num_log_lines()
 end
 
 And(/^The log filtered from bookmark contains messages:$/) do |message|
-  expect(occurences_in_log_filtered_from_bookmark(message)).to eq 1
+  expect(num_matches_since_savepoint(message)).to eq 1
 end
 
-And(/^The log filtered from bookmark "([^"]*)" contains "([^"]*)" messages:$/) do |bookmark, occurrences, message|
-  expect(occurences_in_log_filtered_from_bookmark(bookmark, message)).to eq occurrences.to_i
+And(/^The log filtered from bookmark "([^"]*)" contains "([^"]*)" messages:$/) do |occurrences, message|
```

Comment:
IIRC we don't use this form.  We're only ever checking "number of matches since the last save."  So this one can be deleted.